### ISSUE TITLE
[Bug]: After inserting json into the visualization page of the table and clicking submit, the page does not respond

### ISSUE THREAD

**Issue by Huangxiao-mas** [oceanbase/odc#2853]
### ODC version

ODC431

### OB version

oracle4217

### What happened?

After inserting json into the visualization page of the table and clicking submit, the page does not respond
![image](https://github.com/oceanbase/odc/assets/59468957/b107262a-0131-425b-a570-a53f336bd79f)

### What did you expect to happen?

。

### How can we reproduce it (as minimally and precisely as possible)?

```
CREATE TABLE P_order_detail_test (
  "APPL_SEQ" NUMBER(20) NOT NULL ENABLE,
  "CHANNEL_NO" VARCHAR2(50),
  "APP_ID" VARCHAR2(50),
  "ORDER_TYPE" VARCHAR2(10),
  "ORDER_STATUS" VARCHAR2(1),
  "ORDER_INFO" JSON,
  "CHANNEL_TOPIC" VARCHAR2(50),
  "MESSAGE_ID" VARCHAR2(100),
  "CREATE_TIME" DATE DEFAULT sysdate NOT NULL ENABLE,
  "UPDATE_TIME" DATE DEFAULT sysdate NOT NULL ENABLE,
  CONSTRAINT "P_ORDER_DETAIL_TEST" PRIMARY KEY ("APPL_SEQ", "CREATE_TIME")
) COMPRESS FOR ARCHIVE REPLICA_NUM = 3 BLOCK_SIZE = 16384 USE_BLOOM_FILTER = FALSE TABLET_SIZE = 134217728 PCTFREE = 0
 partition by range(CREATE_TIME)
(partition "P1" values less than (TO_DATE(' 2025-01-01 00:00:00', 'SYYYY-MM-DD HH24:MI:SS', 'NLS_CALENDAR=GREGORIAN')),
partition "P2" values less than (TO_DATE(' 2025-07-01 00:00:00', 'SYYYY-MM-DD HH24:MI:SS', 'NLS_CALENDAR=GREGORIAN')),
partition "P3" values less than (TO_DATE(' 2026-01-01 00:00:00', 'SYYYY-MM-DD HH24:MI:SS', 'NLS_CALENDAR=GREGORIAN')),
partition "P4" values less than (TO_DATE(' 2026-07-01 00:00:00', 'SYYYY-MM-DD HH24:MI:SS', 'NLS_CALENDAR=GREGORIAN')),
partition "P5" values less than (TO_DATE(' 2027-01-01 00:00:00', 'SYYYY-MM-DD HH24:MI:SS', 'NLS_CALENDAR=GREGORIAN')),
partition "P6" values less than (TO_DATE(' 2027-07-01 00:00:00', 'SYYYY-MM-DD HH24:MI:SS', 'NLS_CALENDAR=GREGORIAN')),
partition "P7" values less than (TO_DATE(' 2028-01-01 00:00:00', 'SYYYY-MM-DD HH24:MI:SS', 'NLS_CALENDAR=GREGORIAN')),
partition "P8" values less than (TO_DATE(' 2028-07-01 00:00:00', 'SYYYY-MM-DD HH24:MI:SS', 'NLS_CALENDAR=GREGORIAN')),
partition "P9" values less than (TO_DATE(' 2029-01-01 00:00:00', 'SYYYY-MM-DD HH24:MI:SS', 'NLS_CALENDAR=GREGORIAN')),
partition "P10" values less than (TO_DATE(' 2029-07-01 00:00:00', 'SYYYY-MM-DD HH24:MI:SS', 'NLS_CALENDAR=GREGORIAN')),
partition "P11" values less than (TO_DATE(' 2030-01-01 00:00:00', 'SYYYY-MM-DD HH24:MI:SS', 'NLS_CALENDAR=GREGORIAN'))) ;
```
```
{"body":{"data":{"name":"zhangsan","applSeq":1202406196},"action":"order.submit","orderType":"credit","actionType":"new"},"head":{"appId":"aa","reqId":"1234","userId":"","reqTime":1717640528367,"extraInfo":{}}}
```

**Comment by HSunboy** [comment]
fixed in dev-4.3.1

**Comment by sl01388797** [comment]
![Image](https://github.com/oceanbase/odc/assets/143492234/cb2e86ef-4f21-41b2-9dd7-3cd72436abea)

pass
